### ISSUE TITLE
About Page: nav list does not have tab order support

### ISSUE THREAD

**Issue by MedicalTest** [codalab/codalab-competitions#119]
Issue:  About Page: nav list does not have tab order support

Steps:
1.  Launch CodaLab  (http://cl-test1-ubuntu.cloudapp.net/help/about.html)
2.  Click on About (“About CodaLab” is selected by default.
3.  Hit the tab key on your keyboard

Actual result:  nothing (see attached, “.png”)

Expected result:  user should be able to tab through list of topics in Nav list area.

Configuration details
Build date: 8/23/2013, ~1 p.m.
OS: Win7
Browser(s):   all supported
